api/public/v2/report/views.py

```diff
@@ -118,9 +118,9 @@ def filter_report(
                 return Report()
 
         if path and flag:
-            report = report.filter(flags=[flag], paths=[f"{path}*"])
+            report = report.filter(flags=[flag], paths=[f"{path}.*"])
```

Comment:
Could we add a test that makes the distinction bw the previous solution and this one? I'm assuming certain paths didn't use to work but now they do?

Reply:
Sure thing.
All paths used to work, except they may return more results than expected. For example if you have the folders:
`App/Test` and `App/TestTwo`, you search for path filter `"App/Test/"` it should return only the first folder, not both.